### PR TITLE
SUP-639: Briefly describe Refresh VPC connections function

### PR DESCRIPTION
**Pull request by bcrisologo**
# What changed, and why it matters

We have a "Refresh VPC connections" button available inside the VPC Peering connection ID that has no documentation available.

image

I've inquired about this internally and it seems this button functionality is helpful for changes done on the externally VPC peered hosts (i.e. firewall changes) in the event that there's connection issues after changes were made on the externally peered host(s).

### CODE REVIEW

File: docs/platform/howto/manage-vpc-peering.rst [hunk 1]
```diff
@@ -75,4 +75,17 @@ You can enable public internet access for your services by following the :doc:`E
...
 
-Also note that safelisting applies to both internal and external traffic. If you safelist an external IP address and want to keep traffic flowing with the internal (peered) connections, make sure that you safelist the CIDR blocks of the peered networks as well to avoid disruptions to the service.
\ No newline at end of file
+Also note that safelisting applies to both internal and external traffic. If you safelist an external IP address and want to keep traffic flowing with the internal (peered) connections, make sure that you safelist the CIDR blocks of the peered networks as well to avoid disruptions to the service.
+
+Troubleshoot VPC connection issues
+-------------------------------------
+
+If there have been network changes that were made on VPC peered hosts external from Aiven, it can cause issues with routing to your Aiven services hosted in a VPC.  In such scenarios, you can follow the steps below:
```
Comment (wojcik-dorota) [inline docs/platform/howto/manage-vpc-peering.rst:83]:
```
Any network changes to VPC peered hosts external from Aiven can cause issues with routing to your Aiven services hosted in a VPC. To troubleshoot such issues, take the following steps:
```

File: docs/platform/howto/manage-vpc-peering.rst [hunk 2]
```diff
@@ -75,4 +75,17 @@ You can enable public internet access for your services by following the :doc:`E
...
+
+1. In the Aiven console, click **VPC**.
+2. Locate the VPC ID that is affected and click on the appropriate ID under **Internal ID** column.
+3. Click **Refresh VPC connections** button.
+
+Performing the above will have the platform check the VPC peering connection and rebuild the peering connection state if there are any changes detected.
```
Comment (wojcik-dorota) [inline docs/platform/howto/manage-vpc-peering.rst:89]:
```
As a result, the platform checks the VPC peering connection and rebuilds the peering connection state if there are any changes detected.
```

File: docs/platform/howto/manage-vpc-peering.rst [hunk 3]
```diff
@@ -75,4 +75,17 @@ You can enable public internet access for your services by following the :doc:`E
...
-Also note that safelisting applies to both internal and external traffic. If you safelist an external IP address and want to keep traffic flowing with the internal (peered) connections, make sure that you safelist the CIDR blocks of the peered networks as well to avoid disruptions to the service.
\ No newline at end of file
+Also note that safelisting applies to both internal and external traffic. If you safelist an external IP address and want to keep traffic flowing with the internal (peered) connections, make sure that you safelist the CIDR blocks of the peered networks as well to avoid disruptions to the service.
+
+Troubleshoot VPC connection issues
+-------------------------------------
+
+If there have been network changes that were made on VPC peered hosts external from Aiven, it can cause issues with routing to your Aiven services hosted in a VPC.  In such scenarios, you can follow the steps below:
+
+1. In the Aiven console, click **VPC**.
```
Comment (wojcik-dorota) [inline docs/platform/howto/manage-vpc-peering.rst:85]:
```
1. In `Aiven Console <https://console.aiven.io/>`_, select **VPC**.
```

File: docs/platform/howto/manage-vpc-peering.rst [hunk 4]
```diff
@@ -75,4 +75,17 @@ You can enable public internet access for your services by following the :doc:`E
...
+2. Locate the VPC ID that is affected and click on the appropriate ID under **Internal ID** column.
+3. Click **Refresh VPC connections** button.
```
Comment (wojcik-dorota) [inline docs/platform/howto/manage-vpc-peering.rst:87]:
```
3. Select **Refresh VPC connections**.
```

File: docs/platform/howto/manage-vpc-peering.rst [hunk 5]
```diff
@@ -75,4 +75,17 @@ You can enable public internet access for your services by following the :doc:`E
...
\ No newline at end of file
+Also note that safelisting applies to both internal and external traffic. If you safelist an external IP address and want to keep traffic flowing with the internal (peered) connections, make sure that you safelist the CIDR blocks of the peered networks as well to avoid disruptions to the service.
+
+Troubleshoot VPC connection issues
+-------------------------------------
+
+If there have been network changes that were made on VPC peered hosts external from Aiven, it can cause issues with routing to your Aiven services hosted in a VPC.  In such scenarios, you can follow the steps below:
+
+1. In the Aiven console, click **VPC**.
+2. Locate the VPC ID that is affected and click on the appropriate ID under **Internal ID** column.
```
Comment (wojcik-dorota) [inline docs/platform/howto/manage-vpc-peering.rst:86]:
```
2. Find the ID of the affected VPC and select it from the **Internal ID** column.
```

**Review by wojcik-dorota — CHANGES_REQUESTED** [review]

**Comment by bcrisologo** [comment]
Let me know if the changes look good @wojcik-dorota 🙂

**Review by wojcik-dorota — APPROVED** [review]

**Pull request merged**